### PR TITLE
Remove the bulk version of the time travelling loader

### LINKED ISSUE #46: NCBI taxa bulk loader doesn't include a release timestamp
... unlike the delta loader. That needs to be fixed before the bulk loader (which is used to load the first version of the NCBI taxa tree) is used again.

But first - see how much faster the bulk loader is over the delta loader. If it's not much faster just retire the bulk loader.

### PR DESCRIPTION
**Pull request by MrCreosote**
Although it's ~3x faster than the delta loader, it's only useful for the very
first load. For NCBI taxa (which is the only data set that currently has a
bulk loader implemented) this means a one time time savings of ~7-15 minutes.

See performance_ncbi_bulk_vs_delta_loader.md for more details.

Closes #46

### CODE REVIEW

File: snippets/performance_ncbi_bulk_vs_delta_loader.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ continued support.
   exclusively use the delta loader.
   * They are mostly orders of magnitude smaller than the NCBI dataset, in which case loading with
     the delta loader is fast.
-* As of this writing the NCBI delta loader doesn't support release timestamps or registering
+* As of this writing the NCBI bulk loader doesn't support release timestamps or registering
```
Comment (dauglyon) [inline snippets/performance_ncbi_bulk_vs_delta_loader.md:17]: Why this change? Or was it just previously wrong?

File: relation_engine/batchload/test/load_utils_test.py [hunk 2]
```diff
@@ -1,145 +0,0 @@
...
-_MAX_ADB_INT = 9007199254740991
-
-
-def test_process_nodes_noop():
-    outfile = StringIO()
-    process_nodes([], 'ver', 1, outfile)
-    assert outfile.getvalue() == ''
-
-
-def test_process_nodes():
```
Comment (dauglyon) [inline relation_engine/batchload/test/load_utils_test.py:15]: For the deleted test files, are there equivalent tests for the delta loader where needed? delta_load_integration_test.py seems to be it but just thought I'd ask to double check.

File: snippets/performance_ncbi_bulk_vs_delta_loader.md (hunk 1 above)
Comment (MrCreosote) [reply snippets/performance_ncbi_bulk_vs_delta_loader.md:17]: It was wrong, it's the bulk loader that doesn't support those features. The delta loader does

File: relation_engine/batchload/test/load_utils_test.py (hunk 2 above)
Comment (MrCreosote) [reply relation_engine/batchload/test/load_utils_test.py:15]: yes, the big green files in the sunburst are the 2 delta loader core code files https://app.codecov.io/gh/kbase/relation_engine_importers
